supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -81,17 +81,28 @@
 
 *See also*:
 
-// Azure: General; kept as is
 [discrete]
 [[dotnet]]
-==== image:images/yes.png[yes] .NET Core (noun)
-*Description*: Microsoft's newest development platform, ".NET Core", is an open source and cross-platform framework for building cloud-based internet-connected applications, such as web apps, IoT apps, and mobile backends. All .NET Core applications can run on .NET Core or on the full .NET Framework. Always refer to it as .NET Core; all other variants are incorrect.
+==== image:images/caution.png[with caution] .NET (noun)
+*Description*: Microsoft's development platform, ".NET", is an open source and cross-platform framework for building cloud-based internet-connected applications, such as web apps, IoT apps, and mobile back ends. All .NET applications can run on .NET or on the .NET Framework. For version 5 and later, refer to it as ".NET". For earlier versions, use ".NET Core".
 
-*Use it*: yes
+*Use it*: with caution
```

Comment:
Just wondering if we should change this to "yes" for the new version and "no" for the old version, instead of having both as "with caution". Other entries that we have newer names for follow this convention.

See "Red Hat JBoss Data Grid" and "Red Hat Data Grid" for an example

If you do make a change, be sure to change the icons as well

Reply:
Changed.